miden/Cargo.toml

```diff
@@ -57,6 +57,7 @@ metal = ["prover/metal", "std"]
 std = ["assembly/std", "processor/std", "prover/std", "verifier/std"]
 
 [dependencies]
+air = { package = "miden-air", path = "../air", version = "0.10", default-features = false }
```

Comment:
I'm wondering we should re-export `RowIndex` from the `processor` crate. If we do that, this dependency wouldn't be need, I think.